### ISSUE TITLE
Kneeboard Enhancements

### ISSUE THREAD

**Issue by walterroach** [dcs-liberation/dcs_liberation#338]
- [ ] Add package frequencies to kneeboard.  
  * This would be useful for listening for the status of other flights in your package.
- [ ] Add a page with target picture and description for strike flights
  * The same picture and description used in the UI would be fine, would help with target identification when flying strike.
- [ ] Add heading to each waypoint.

**Comment by walterroach** [comment]
Also related to #331

**Comment by DanAlbert** [comment]
Target information is tough because we can't reliably convert X/Y to lat/lon, nor do we have altitude information.

I'd bet we could do the former, and maybe DCS maps are accurate enough that we could get the altitudes online? I think inaccurate data would be worse than no data though, since bad data means the JDAM will miss.

**Comment by walterroach** [comment]
I don't mean to add detailed target location data, just the simple description of the units and the encyclopedia pictures we put in the UI.

![image](https://user-images.githubusercontent.com/37820425/98598979-e2326f80-22a0-11eb-96dd-03734f6b3029.png)

**Comment by HILOK** [comment]
hi guys, happy new year!

here's an example how the kneeboard could be organized on 1 page. not claiming this to be the best solution, but so far it worked well for me.

[COMBAT DATA CARD.pdf](https://github.com/Khopa/dcs_liberation/files/5761344/COMBAT.DATA.CARD.pdf)

notes:
-on the top right are bullseye coordinates
-all information taken from the ME ingame

here is the word doc
[COMBAT DATA CARD.docx](https://github.com/Khopa/dcs_liberation/files/5761346/COMBAT.DATA.CARD.docx)

**Comment by rabauke80** [comment]
Also, how about a dedicated target page in case you have a longer list of strike targets. Currently, they get summed up, which makes ad-hoc prioritization hard

current:
![image](https://user-images.githubusercontent.com/67376231/104118357-b6774880-5328-11eb-89ef-9a7b59b04652.png)

- [ ] add task type to heading (e.g. STRIKE, SEAD, DEAD, ...)
- [ ] add additional dedicated target page - gives additional space for target details and remarks and keeps the front page tidy

**Comment by jeetsukumaran** [comment]
Just a note that even (just) Lon/Lat for targets in the kneeboard (in multiple formats) would be great. 

ESPECIALLY (and maybe only) for OCA runway strikes.

For programming JDAM's for stationary ground units or static units, it's easy to pull from the F-10 map: just need to click the icon. But when trying to strike airbase runways, clicking the icon gives the coordinates for an area at the edge of the airbase. JDAM's/JSOW's here do no damage to the runway. 

Now, we can hover the mouse over the runway and get the info by looking up at the status bar ..... but this is phenomenally insanely a PIA to do in VR! Because every movement of your head moves the cursor. You have to zoom in really far, and then shift the map so the runway point is very close to the top of your view, then try to keep your head as perfectly still as possible and use peripheral vision to read off the coordinates, memorize as many figures as you can, and then type them in.  Of course, switching back and forth from the cockpit view to the map view totally throws off everything. So there is a lot of back and forth.

We _do_ have a strike waypoint with the coordinates, but I don't think that this is in the correct format for the JDAM/JSOW weapon. Of course, it can be converted by using an external calculator, but again in VR it is a massive PIA.

Elevation of the target is much more easily available and retrievable: just one number, and the whole airfield (or rnyway at least) typically has the same elevation within reasonable margin or error so can just read off and type.

**Comment by walterroach** [comment]
> but this is phenomenally insanely a PIA to do in VR! 

@jeetsukumaran 
I agree coordinates would be useful.  Check this add-on out though, hugely helpful for taking notes in VR. https://github.com/rkusa/dcs-scratchpad

**Comment by jeetsukumaran** [comment]
Already use it :) 

That certainly solved part of the problem. It's the challenge of being able to obtain the coordinates from the F10 map that remains though.

It's also (now) actually pretty easy in the mission editor (ctrl-Y or something similar calls up a dialog with it). Problem is the ME is not always available (e.g., MP).
